src/ui/layouts/common/layouts/AuthenticatedLayout/AuthenticatedSidebar/Menu/MenuItem.tsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
-import { NavLink, useLocation } from 'react-router-dom';
-import { Paragraph, Box, FlexBox } from '../../../../../../components';
-import cn from 'classnames';
-import styles from './MenuItem.module.scss';
+import { NavLink } from 'react-router-dom';
+
+import styles from './MenuItem.module.css';
```

Comment:
Just suggestion,
If styles import will be at last below all imports arranged properly. Then, It will be easier to find sometimes imports.